pages/index.js

```diff
@@ -1,13 +1,26 @@
-import Link from 'next/link';
+import Button from "@/components/form/Button";
 
 export default function Home() {
   return (
     <>
       <h1 className={styles.header}>Welcome to Amcovad</h1>
+
+      <h2>Solid Button</h2>
+      <Button>Sign in</Button>
+
+      <Button color="secondary">try it now</Button>
+
+      <h2>Outline Button</h2>
+
+      <Button outline>Sign in</Button>
+
+      <Button outline color="secondary">
+        try it now
+      </Button>
     </>
   );
 }
 
 const styles = {
-  header: 'text-3xl font-bold underline text-red-500',
+  header: "text-3xl font-bold underline text-red-500",
```

Comment:
As discussed, you will need to fix your Linting library or Prettier to support single quote for normal strings.

```suggestion
  header: 'text-3xl font-bold underline text-red-500',
```